google-recaptcha.html

```diff
@@ -95,10 +112,10 @@
       },
 
       observers: [
-        '_validate(theme, type, timeout)'
+        '_validateAttrs(theme, type, timeout)'
       ],
 
-      _validate:function () {
+      _validateAttrs: function () {
```

Comment:
Calling this `_validate` is confusing once support for form validation logic is added so I propose renaming it. Let me know if something else (maybe that doesn't even have the word validate in it -- maybe `checkOptions`?) would be better.